src/main/java/com/dkohut/chatservice/common/dao/ChatRoomDAOService.java

```diff
@@ -0,0 +1,200 @@
+package com.dkohut.chatservice.common.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.jtemplate.sql.Parameters;
+import org.springframework.stereotype.Repository;
+
+import com.dkohut.chatservice.common.entity.ChatRoom;
+import com.dkohut.chatservice.common.interfaces.IChatRoomDAOService;
+
+
+/**
+ * This class responsible for executing queries in database and extracting info from it.
+ * 
+ * @author Dmytro Kohut
+ *
+ */
+@Repository
+public class ChatRoomDAOService implements IChatRoomDAOService {
+	
+	private static final String ID = "id";
+	private static final String NAME = "name";
+	private static final String FIRST_ROW = "firstRow";
+	private static final String NUMBER_OF_ROWS = "numberOfRows";
+	
+	private static final String QUERY_SELECT_SPECIFIED_NUMBER_OF_ROWS = "SELECT id, name, created_at FROM chat_rooms ORDER BY "
+												+ "created_at LIMIT :numberOfRows OFFSET :firstRow";
+	private static final String QUERY_SELECT_BY_ID = "SELECT id, name, created_at FROM chat_rooms WHERE id=:id";
+	private static final String QUERY_CREATE = "INSERT INTO chat_rooms(name, created_at) VALUES(:name, NOW())";
+	private static final String QUERY_UPDATE = "UPDATE chat_rooms SET name=:name WHERE id=:id";
+	private static final String QUERY_DELETE = "DELETE FROM chat_rooms WHERE id=:id";
+	
+	private static final Logger logger = Logger.getLogger(ChatRoomDAOService.class);
+	
+	private static Connection connection;
+	
+	
+	static {
+		try {
+			Class.forName("com.mysql.jdbc.Driver");
+			connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/chats", "root", "root");
```

Comment:
It would be better to use configuration file instead of local settings.

Reply:
@MaximusFk, I can't read values from application.properties or another configuration file directly into static fields using `@value`. Instead of I can use the non-static setter for this purpose but I'm not sure it is a good idea.

Reply:
@finr1r, You can try to create class with logic creating connection, inside you can define @value fields and in DAOServices use @Autowired field.:)

Reply:
@MaximusFk, the way you recommended me is a creating for each object own connection, the way that I used create connection once - when class initialized and used it in each object and each query. Both your and my ways have advantages and disadvantages, but I actually like the idea about new class with database connection logic, I will make it.